content/release-10-18-0/whatsnew-10-18-0.md

```diff
@@ -7,6 +7,39 @@ layout: bundle
 Release 10.18.0 includes the following new features or major feature enhancements.
 
 ### Application enablement
+#### Global Time Selector for dashboards
+
+A new global time context feature has been introduced, allowing users to apply a consistent time context (e.g., display data from the last hour, last month, etc.) across an entire dashboard. Widget developers can now set the global time context for the whole dashboard or a particular widget. Widgets bound to the global time context are marked accordingly. Users can also share dashboards with a specific time context by including the time context parameters in a URL.
+
+The global time context is available for the following existing widgets: Event list, Data points graph, and Data points table.
+
+#### Map feature migration and enhancements
+
+The map feature has been migrated from AngularJS to Angular, bringing along several improvements and modifications:
+
+Breaking changes:
+* Real-time tracking has been limited to a single device at a time.
+  
+New features:
+
+* A device can now be followed in real-time.
+* Device icons can be customized.
+* Zoom level adjustments have been introduced.
+* Center point settings have been added.
```

Comment:
Is that clear to everybody what that means? Could we rephrase this a bit?

Reply:
what about:

Center point settings have been added to the map, allowing users to designate a specific point for the map to consistently center on. Users can also utilize their current location as the central point. The map will remain centered on this point when refreshing the page or navigating back from another tab.